main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+const digits = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+func toBase10(num string, fromBase int) int {
+	ans, _ := strconv.ParseInt(num, fromBase, 0)
```

Comment:
It's good that you found the package `strconv`, but this is not a solution that we expected. You should have implemented the algorithm that we were using during a lesson on a paper.